pkg/manager/router/router.go

```diff
@@ -30,9 +30,12 @@ import (
 type Router interface {
 	Route(RedirectableConn) (string, error)
 	RedirectConnections() error
+	HasConns() bool
```

Comment:
I prefer `ConnCount()` to count the connections. This interface will be more general.